samples/v1.0/Elements/Column.SelectAction.json

```diff
@@ -25,7 +25,7 @@
 					],
 					"selectAction": {
 						"type": "Action.OpenUrl",
-						"title": "cool link",
+						"tooltip": "cool link",
```

Comment:
>"tooltip": "cool link", [](http://example.com/codeflow?start=6&length=23)

should we modify previous versions? we didn't do it with Input.* elements so we should file bugs in if we're set on following this path

Reply:
I do believe we should be on that path. See https://github.com/microsoft/AdaptiveCards/issues/5653.